masterlist.yaml

```diff
@@ -2918,16 +2854,20 @@ plugins:
         - lang: ja
           text: 'このファイルをアンインストールしてください。Homemaker 1.50以降では不要です。'
         condition: 'version("Homemaker.esm", "1.50", >=)'
+
   - name: 'SettlementKeywords.esm'
     url: [ 'https://www.nexusmods.com/fallout4/mods/12226/' ]
+    req:
+      - name: '../Fallout4.exe'
+        display: 'Fallout 4 1.6.3+'
+        condition: 'version("../Fallout4.exe", "1.6.3", <)'
```

Comment:
I'm pretty sure conditions with version or checksums don't work with `req:` keys.
I've had to use `&requiresX` instead.

Reply:
Thanks for the heads up. I'll double-check just in case.